sql/collations.go

```diff
@@ -152,8 +152,8 @@ const (
 	Collation_ucs2_lithuanian_ci          CollationID = 140
 	Collation_ucs2_persian_ci             CollationID = 144
 	Collation_ucs2_polish_ci              CollationID = 133
-	Collation_ucs2_roman_ci               CollationID = 143
 	Collation_ucs2_romanian_ci            CollationID = 131
+	Collation_ucs2_roman_ci               CollationID = 143
```

Comment:
I see that we added new collations, but why were the pre-existing collations moved around?

Reply:
Ah forgot to put back the moved around ones when I was matching it with MySQL result to track each of them.